packages/gatsby/src/components/details/Contributors.js

```diff
@@ -1,16 +1,23 @@
+import styled  from '@emotion/styled';
 import React   from 'react';
 
 import {Owner} from '../hit';
 
+const List = styled`
```

Comment:
I think the element name might be missing?
```suggestion
const List = styled.ul`
```

Reply:
you're right, seems like I forgot to save my file before committing :)